test/WebJobs.Extensions.RabbitMQ.Samples/RabbitMQSamples.cs

```diff
@@ -106,6 +106,14 @@ public static class RabbitMQSamples
             logger.LogInformation($"RabbitMQ queue trigger function processed message: {pocObj}");
         }
 
+        public static void RabbitMQTrigger_JsonToPOCO(
+            [RabbitMQTrigger(connectionStringSetting: "rabbitMQ", "new_test_queue-poison", "dlxName")] string res,
+            ILogger logger)
+        {
+            logger.LogInformation($"RabbitMQ queue trigger function processed message: {res}");
+        }
+

```

Comment:
So this sample actually doesn't work. :( Headers are automatically added to deadlettered messages so configuring "dlxName" as a header won't match the headers of the poisoned message. Let me know what you think! @pragnagopa

Reply:
Add documentation in the Summary that explains trigger is setup on poison queue from function 'RabbitMQTrigger_JsonToPOCO' , also rename  `RabbitMQTrigger_JsonToPOCO_DeadLetter` to RabbitMQTrigger_Process_PoisonQueue